docs/resiliency/index.asciidoc

```diff
@@ -134,12 +123,36 @@ We have increased our test coverage to include scenarios tested by Jepsen. We ma
 This status page is a start, but we can do a better job of explicitly documenting the processes at work in Elasticsearch, and what happens in the case of each type of failure. The plan is to have a test case that validates each behavior under simulated conditions. Every test will document the expected results, the associated test code and an explicit PASS or FAIL status for each simulated case.
 
 [float]
-=== Do not allow stale shards to automatically be promoted to primary (STATUS: ONGOING)
+=== Make index creation more user friendly (STATUS: ONGOING)
```

Comment:
I think this issue is actually about "Make index creation resilient to index closing and a full cluster crash"

The description should explain the issue where we used to require a quorum (with an exception for 2) of shards to recover a primary which means that if a cluster dies before enough shards have been allocated the primary could not be assigned. SImilarly, since index creation doesn't wait on all shards to be assigned. One could close an index before enough shard copies were started. This made the index impossible to open.